### PR TITLE
Update Organization UI

### PR DESCRIPTION
**Pull request by mmjee**
JIRA Ticket: https://team-1624093970686.atlassian.net/browse/AR-737?atlOrigin=eyJpIjoiYjE2ZGViMGJjNDI2NGY1OWJhYjE2ZTY5YTAxNzljOGYiLCJwIjoiaiJ9

### CODE REVIEW

**Review by shrinathprabhu — APPROVED** [review]
Looks good. Just curious, why are those extra spaces/tabs added in html

**Comment by mmjee** [comment]
> Looks good. Just curious, why are those extra spaces/tabs added in html

3 lines were incorrectly indented (just missing spaces on the left), so I thought of fixing them while I was in the area.

File: src/pages/AppProfile.vue [hunk 1]
```diff
@@ -126,37 +126,20 @@
...
-              ></div>
-            </div>
-            <div class="flex flex-wrap" style="justify-content: space-between">
               <div class="flex column details">
                 <span class="body-2">Country</span>
                 <span
-                  class="sub-heading-3 overflow-ellipsis"
-                  :title="organisationDetails.country"
-                  v-if="!editOrganisationDetails"
+                    class="sub-heading-3 overflow-ellipsis"
```
Comment (ajithranka) [inline src/pages/AppProfile.vue:132]: Please check the indentation here and in the hidden div below.

Comment (mmjee) [reply src/pages/AppProfile.vue]: Just realized what's going on, we should add a `.editorconfig` file.

**Review by ajithranka — APPROVED** [review]

**Pull request merged**